test/unit/test_Model.cc

```diff
@@ -156,6 +156,41 @@ TEST_F(UFModelBuilderTest, test_functionModel) {
     ASSERT_TRUE(logic.isIte(templateFun.getBody()));
 }
 
+class UFConstModelTest : public ::testing::Test {
+protected:
+    UFConstModelTest() : logic(), ms(logic, c, "uf-solver") {
+        char * msg;
+        SRef U = logic.declareSort("U", &msg);
+        SymRef fs = logic.declareFun("f", U, {U}, &msg, false);
+        x = logic.mkVar(U, "x");
+        a = logic.mkConst(U, "a");
+        f_a = logic.mkUninterpFun(fs, {x});
```

Comment:
```suggestion
        f_a = logic.mkUninterpFun(fs, {a});
```

With this change, the unit test shows the problem I mentioned.

Reply:
Fixed in the below commit, thanks!